src/App.tsx

```diff
@@ -1,15 +1,19 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.scss';
-import SignIn from './pages/signIn/SignIn';
+import React from "react";
+
+import logo from "./logo.svg";
+import "./App.scss";
+
+import SignIn from "./pages/signIn/SignIn";
```

Comment:
I think better change path. You should create index file in folder ```signIn``` and import there your component ```SignIn``` and ```scss``` and after you can use this path for import ```import SignIn from "./pages/signIn/```